src/main/java/org/prebid/server/currency/CurrencyConversionService.java

```diff
@@ -153,34 +152,30 @@ public BigDecimal convertCurrency(BigDecimal price, Map<String, Map<String, BigD
     }
 
     /**
-     *  Get conversion rate from request currency rates if it is present
+     *  Get conversion rate from request currency rates if it is present and then from server if it is not processed.
      */
-    private static BigDecimal getConversionRateFromRequest(Map<String, Map<String, BigDecimal>> externalCurrencyRates,
-                                                           Map<String, Map<String, BigDecimal>> requestCurrencyRates,
-                                                           String adServerCurrency, String effectiveBidCurrency) {
+    private BigDecimal getConversionRateFromRequest(Map<String, Map<String, BigDecimal>> requestCurrencyRates,
+                                                         String adServerCurrency, String effectiveBidCurrency) {
         BigDecimal conversionRate = null;
         if (requestCurrencyRates != null) {
```

Comment:
you can omit this check (or add similar one in getConversionRateFromServer, be consistent)